app/models/token.rb

```diff
@@ -14,4 +16,10 @@ def name
   define_method(:image) do
     token_image.bind(self).().presence || tokenable.try(:image)
   end
+
+  private
+
+  def broadcast_token_creation
+    TokenBroadcastJob.perform_later(self)
```

Comment:
Does this need to happen in a job? We don't currently have a proper job queue setup yet.

Reply:
It makes sense for it to be a job so that it doesn't block the request. All the examples of this online put it in a job. 

[Rails by default comes with an asynchronous queuing implementation that runs jobs with an in-process thread pool. Jobs will run asynchronously, but any jobs in the queue will be dropped upon restart.](https://edgeguides.rubyonrails.org/active_job_basics.html#the-purpose-of-active-job)

We can eventually add a more robust background execution mechanism, but I don't think it's necessary to do right now/in this PR.